apps/patient/src/components/PharmacyCard.tsx

```diff
@@ -33,7 +33,7 @@ const RatingHours = ({ businessStatus, rating, hours }) => {
 
   if (!rating || !hours) return null
 
-  const { open, is24Hr, opens, closes } = hours
+  const { open, is24Hr, opens, opensDay, closes } = hours
```

Comment:
can this be typed? figured it'd be barking since it's not in here https://github.com/Photon-Health/client/blob/4026031e393f152b05bd7a40cbbb82fb06272b59/apps/patient/src/utils/models.ts#L38

Reply:
ah! wonder why it didn't, nice catch

Reply:
added